ocfweb/account/print.py

```diff
@@ -0,0 +1,147 @@
+import base64
+import os.path
+import uuid
+from io import BytesIO
+
+import qrcode
+from django import forms
+from django.forms import widgets
+from django.http import HttpRequest
+from django.http import HttpResponse
+from django.shortcuts import render
+from paramiko import AuthenticationException
+from paramiko import SSHClient
+from paramiko.hostkeys import HostKeyEntry
+
+from ocfweb.component.forms import Form
+
+PRINT_FOLDER = '.user_print'
+
+
+def print(request: HttpRequest) -> HttpResponse:
+    if request.method == 'POST':
+        form = PrintForm(
+            request.POST,
+            request.FILES,
+            initial={
+                'double_or_single': 'single',
+            },
+        )
+        qr_b64 = b''
+        double_or_single = 'single'
+        error = ''
+        if form.is_valid():
+            username = form.cleaned_data['username']
+            password = form.cleaned_data['password']
+
+            file = request.FILES['file']
+
+            double_or_single = form.cleaned_data['double_or_single']
+
+            ssh = SSHClient()
+
+            host_keys = ssh.get_host_keys()
+            entry = HostKeyEntry.from_line(
```

Comment:
Would prefer we refactor this to share logic with https://github.com/ocf/ocfweb/blob/8b35c17d91a7d722882bf2672c35099e22e573e6/ocfweb/account/commands.py but not a PR blocker

Reply:
Yeah I will do this in a followup.